packages/devtools-ui-kit/src/components/NDialog.vue

```diff
@@ -53,7 +53,7 @@ export default {
   <Teleport v-if="shown" to="body">
     <div
       v-show="show"
-      class="n-dialog fixed inset-0 z-100 flex items-center justify-center n-transition"
+      class="ignore-click-outside n-dialog fixed inset-0 z-100 flex items-center justify-center n-transition"
```

Comment:
when a drawer is open and has auto-close, if a dialog popsup and when user click on it, it would close the drawer and dialog, this way we can add `ignore-click-outside` to ignore list so it doesn't close.

ps: should we rename the `ignore-click-outside` to something like `nuxt-devtools-dialog`?

Reply:
I am fine keeping that name for explicitness

Reply:
@antfu thanks